### ISSUE TITLE
request.url encode

### ISSUE THREAD

**Issue by xghrbc1001** [ariya/phantomjs#13753]
code as below:
```
server.listen(port, function (request, response) {
    var url='';
    if(host.length <= 0){
        url = "http://www.xxx.com"
        + request.url;
```
but while my url contains chinese character,like '百度‘, url encode is '%E7%99%BE%E5%BA%A6',angularjs will send '%E7%99%BE%E5%BA%A6' to phantomjs,but we found the request.url change to '%C3%A7%C2%99%C2%BE%C3%A5%C2%BA%C2%A6',what config we can use that  phantomjs can send  '%E7%99%BE%E5%BA%A6' to my server,not  '%C3%A7%C2%99%C2%BE%C3%A5%C2%BA%C2%A6'

**Comment by zackw** [comment]
This _may_ be fixed by the patches on the [encode-all-the-urls branch](https://github.com/zackw/phantomjs/tree/encode-all-the-urls).  If you can, please test that branch.  I can hook you up with Linux or OSX binaries if you need them, but if you need Windows you're on your own.

**Comment by xghrbc1001** [comment]
@zackw   now it effects by the aid of nodejs,thank you

**Comment by zackw** [comment]
@xghrbc1001 I'm sorry, I do not understand what you are trying to tell me.

**Comment by zackw** [comment]
I believe this is another manifestation of bug #12216.  If so, that is fixed by pull request #12233, which will be included in the upcoming 2.0.1 release.  If you are able to build and test the development sources and confirm whether your problem is solved, that would be helpful.  I am going to close this bug report now, but if your problem is _not_ solved, feel free to reopen it.
